tests/testthat/test-rename_at.R

```diff
@@ -40,7 +40,8 @@ test_that("mulitple functions", {
 
 test_that("no input", {
   # Wait for call pass through
```

Comment:
Is this still valid? if this refers to `step_mutate_at()` showing up in the error message, then we can remove this.

Reply:
good eye! those are no longer needed